app/scripts/proactive/view/ViewWithProperties.js

```diff
@@ -62,7 +62,8 @@ define(
 					} else {
 						jobBreadcrumb = $('<li class="active"><span id="breadcrumb-job-name">' + modelInUse.get("Name") + '</span></li>');
 						breadcrumb.append(jobBreadcrumb)
-
+                        //hidden the context-menu
+                        $(".context-menu-canvas").hide();

```

Comment:
Why the indentation is not at the same level ?

Reply:
I don't know why.

Reply:
Can you fix it ?

Reply:
I couldn't because I don't have this in intellij